bpm/bonita-core/bonita-process-engine/src/test/java/org/bonitasoft/engine/api/impl/transaction/actor/ImportActorMappingTest.java

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright (C) 2015 BonitaSoft S.A.
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ ******************************************************************************/
+
+package org.bonitasoft.engine.api.impl.transaction.actor;
+
+import org.bonitasoft.engine.actor.mapping.ActorMappingService;
+import org.bonitasoft.engine.actor.mapping.model.impl.SActorImpl;
+import org.bonitasoft.engine.bpm.bar.actorMapping.Actor;
+import org.bonitasoft.engine.bpm.bar.actorMapping.ActorMapping;
+import org.bonitasoft.engine.commons.exceptions.SBonitaException;
+import org.bonitasoft.engine.identity.IdentityService;
+import org.bonitasoft.engine.identity.model.impl.SGroupImpl;
+import org.bonitasoft.engine.identity.model.impl.SRoleImpl;
+import org.bonitasoft.engine.identity.model.impl.SUserImpl;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by mazourd on 03/07/15.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ImportActorMappingTest {
+
+    @Mock
+    private ActorMappingService actorMappingService;
+    @Mock
+    private IdentityService identityService;
+    @Mock
+    private ActorMapping actorMapping;
+    @InjectMocks
+    private ImportActorMapping importActorMapping;
+    private long ACTOR_ID = 12;
+
+    @Test
+    public void UTest_execute_method() throws SBonitaException {
```

Comment:
I don't understand the purpose of this test, you should name correctly your test accordingly to what BEHAVIOR is tested